php/php.editor/src/org/netbeans/modules/php/editor/verification/IncorrectNamedArgumentsHintError.java

```diff
@@ -0,0 +1,195 @@
+    }
+
+    @Override
+    @NbBundle.Messages({
+        "# {0} - name",
+        "IncorrectNamedArguments.desc.duplicate.name=Duplicate name: \"{0}\" already exists.",
+        "IncorrectNamedArguments.desc.combine.named.argument.and.argument.unpacking=Can't combine named arguments(name: arg) and argument unpacking(...)",
+        "IncorrectNamedArguments.desc.positional.arguments.after.named.argument=Can't use positional argument after named argument.",
+    })
+    public void invoke(PHPRuleContext context, List<Hint> hints) {
+        PHPParseResult phpParseResult = (PHPParseResult) context.parserResult;
+        if (phpParseResult.getProgram() == null) {
+            return;
+        }
+        FileScope fileScope = context.fileScope;
+        fileObject = phpParseResult.getSnapshot().getSource().getFileObject();
+        if (fileScope != null && fileObject != null) {
+            if (CancelSupport.getDefault().isCancelled()) {
+                return;
+            }
+            CheckVisitor checkVisitor = new CheckVisitor();
+            phpParseResult.getProgram().accept(checkVisitor);
+            for (NamedArgument argument : checkVisitor.getDuplicateNames()) {
+                if (CancelSupport.getDefault().isCancelled()) {
+                    return;
+                }
+                addHint(argument, Bundle.IncorrectNamedArguments_desc_duplicate_name(argument.getParameterName().getName()), hints);
+            }
+            for (Map.Entry<NamedArgument, Variadic> entry : checkVisitor.getCombinedNamedArgumentsWithArgumentUnpacking().entrySet()) {
+                if (CancelSupport.getDefault().isCancelled()) {
+                    return;
+                }
+                addHint(entry.getKey(), Bundle.IncorrectNamedArguments_desc_combine_named_argument_and_argument_unpacking(), hints);
+                addHint(entry.getValue(), Bundle.IncorrectNamedArguments_desc_combine_named_argument_and_argument_unpacking(), hints);
+            }
+            for (Expression argument : checkVisitor.getArgumentsAfterNamedArgument()) {
+                if (CancelSupport.getDefault().isCancelled()) {
+                    return;
+                }
+                addHint(argument, Bundle.IncorrectNamedArguments_desc_positional_arguments_after_named_argument(), hints);
+            }
+        }
+    }
+
+    private void addHint(ASTNode node, String description, List<Hint> hints) {
+        addHint(node, description, hints, Collections.emptyList());
+    }
+
+    private void addHint(ASTNode node, String description, List<Hint> hints, List<HintFix> fixes) {
+        hints.add(new Hint(
+                this,
+                description,
+                fileObject,
+                new OffsetRange(node.getStartOffset(), node.getEndOffset()),
+                fixes,
+                500
+        ));
+    }
+
+    //~ Inner classes
+    private static final class CheckVisitor extends DefaultVisitor {
```

Comment:
There is missing `visit(ClassInstanceCreation node)`, so this code is not marked with hint (duplicate arguments):
```php
$date = new \DateTime(datetime: 'now', datetime: 'now');
`

Reply:
Will fix. Thanks.